AsyncDisplayKit/ASDisplayNode+Subclasses.h

```diff
@@ -472,6 +494,39 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)asyncTraitCollectionDidChange;
 
+/**
+ * @abstract Called whenever the visiblity of the node changed.
+ *
+ * @discussion Subclasses may use this to monitor when they become visible.
+ */
+- (void)visibilityDidChange:(BOOL)isVisible ASDISPLAYNODE_REQUIRES_SUPER ASDISPLAYNODE_DEPRECATED;
```

Comment:
@garrettmoon yeah, that seems best.  They can stay in the same file at the bottom.